projects/storefrontlib/src/cms-components/product/product-list/product-facet-navigation/product-facet-navigation.component.html

```diff
@@ -7,75 +7,7 @@
   {{ 'productList.filterBy.label' | cxTranslate }}
 </button>
 
-    <ng-container *ngIf="visibleFacets$ | async as visibleFacets">
-      <ng-container *ngFor="let facet of visibleFacets; let facetId = index">
-        <div class="cx-facet-group">
-          <div class="cx-facet-header">
-            <a
-              class="cx-facet-header-link"
-              (click)="toggleFacet(facet.name)"
-              [attr.aria-expanded]="!isFacetCollapsed(facet.name)"
-              aria-controls=""
-              tabindex="0"
-            >
-              {{ facet.name }}
-              <cx-icon
-                [type]="
-                  isFacetCollapsed(facet.name)
-                    ? iconTypes.EXPAND
-                    : iconTypes.COLLAPSE
-                "
-              ></cx-icon>
-            </a>
-          </div>
-          <ng-container *ngIf="!isFacetCollapsed(facet.name)">
-            <ul class="cx-facet-list">
-              <li
-                *ngFor="
-                  let value of getVisibleFacetValues(facet);
-                  index as facetValueId
-                "
-              >
-                <div class="form-check">
-                  <label class="form-checkbox cx-facet-label">
-                    <input
-                      class="form-check-input cx-facet-checkbox"
-                      role="checkbox"
-                      type="checkbox"
-                      aria-checked="true"
-                      [checked]="value.selected"
-                      (change)="toggleValue(value.query.query.value)"
-                    />
-                    <span class="cx-facet-text"
-                      >{{ value.name }} ({{ value.count }})</span
-                    >
-                  </label>
-                </div>
-              </li>
-              <li
-                class="cx-facet-toggle-btn cx-action-link"
-                (click)="showLess(facet.name)"
-                *ngIf="showAllPerFacetMap.get(facet.name)"
-              >
-                {{ 'productList.showLess' | cxTranslate }}
-              </li>
-              <li
-                class="cx-facet-toggle-btn cx-action-link"
-                (click)="showMore(facet.name)"
-                *ngIf="
-                  !showAllPerFacetMap.get(facet.name) &&
-                  facet.values.length > facet.topValueCount
-                "
-                tabindex="0"
-              >
-                {{ 'productList.showMore' | cxTranslate }}
-              </li>
-            </ul>
-          </ng-container>
-        </div>
-      </ng-container>
-    </ng-container>
-  </div>
+<cx-active-facets></cx-active-facets>
```

Comment:
This looks wrong. The facet navigation was refactored heavily, you probably need to merge it but not override anything.

Reply:
@tobi-or-not-tobi I'm aware of the changes, if i compare this file with `develop` and with the branch where you made those changes, all 3 files look the same.

Reply:
![Screen Shot 2020-04-20 at 9 38 55 AM](https://user-images.githubusercontent.com/15113219/79758229-1f2a1b00-82eb-11ea-9d9b-46e401061564.png)